### PR TITLE
Feature/ email registration on course for old users

### LINKED ISSUE #354: Send email when a user is registered to a course.
**Is your feature request related to a problem? Please describe.**
Right now, the `register_students` plugin only sends an email to the students when they are registered to UNCode but not to the course. Thus, this would be clearer to the students to inform them when they are enrolled to the course.

**Describe the solution you'd like**
Send an email saying the course they were enrolled to and a bit more of information about how to proceed to access the course and information for the user in general.

### PR DESCRIPTION
**Pull request by VirtualDiegox**
# Description

This is a new feature that sends an email to old users when they're enrolled on a course.

Fixes #354 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- It was tested by checking the email of 3 accounts added via csv and verifing which email is received depending if the user is already registered or not

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: inginious/frontend/i18n/es/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -3088,56 +3086,120 @@ msgstr ""
...
+msgid "How to proceed?"
+msgstr "¿Cuál es el siguiente paso?"
+
+#: inginious/frontend/plugins/register_students/static/email_course_registration_template.html:15
+msgid ""
+"When you logged in, you could see the main page with the courses that you"
+" are enrolled"
+msgstr ""
+"Cuando ingreses podrás ver la página principal con los cursos a los "
+"cuales estas inscrito."
```
Comment (ferestrepoca) [inline inginious/frontend/i18n/es/LC_MESSAGES/messages.po]:
```suggestion
"cuales estás inscrito."
```

File: inginious/frontend/i18n/es/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -3088,56 +3086,120 @@ msgstr ""
...
+#: inginious/frontend/plugins/register_students/static/email_template.html:19
 msgid "How to change the password?"
 msgstr "¿Cómo cambiar la contraseña?"
 
-#: inginious/frontend/plugins/register_students/static/email_template.html:19
-#: inginious/frontend/plugins/register_students/static/email_template.html:26
+#: inginious/frontend/plugins/register_students/static/email_template.html:21
+#: inginious/frontend/plugins/register_students/static/email_template.html:28
 msgid "In top nav bar, click on the option displaying your name."
 msgstr "En la barra superior, haga clic dónde su nombre se muestra."
```
Comment (ferestrepoca) [inline inginious/frontend/i18n/es/LC_MESSAGES/messages.po]:
```suggestion
msgstr "En la barra superior, haga clic donde se muestra su nombre."
```

**Review by ferestrepoca — CHANGES_REQUESTED** [review]

**Review by ferestrepoca — APPROVED** [review]

File: inginious/frontend/plugins/register_students/pages/api/add_course_students_csv_file_api.py [hunk 3]
```diff
@@ -120,6 +132,22 @@ def _is_user_email_equal(self, user_email, data_email):
...
+
+    def _email_user_register_on_course(self, username, email, course):
+        """
+        Format an email to an existing user enrolled to a course,
+        :param email: string of the email of the user
+               course: course to enroll
+        :return: a Tuple with the email and the content of the email to send.
+        """
+        data_policy_link = web.ctx.home + "/data_policy"
+        course_link = web.ctx.home + "/course/" + course.get_id() + "/"
```
Comment (AndresFelipeCastellanos) [inline inginious/frontend/plugins/register_students/pages/api/add_course_students_csv_file_api.py]:
```suggestion
        course_link = web.ctx.home + "/course/" + course.get_id()
```

Remove the last character from course_link url, as courses will not be found with it.

File: inginious/frontend/plugins/register_students/static/email_course_registration_template.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<html>
+    <head></head>
+    <body>
+        <h2>$:_("Welcome on {course_name}")</h2>
+        
+        <h4>$:_("Hello {username}")</h4>
```
Comment (AndresFelipeCastellanos) [inline inginious/frontend/plugins/register_students/static/email_course_registration_template.html]: There is no need to translate the parameters in template, just translate text messages and leave the parameters outside the parentheses.

File: inginious/frontend/plugins/register_students/static/email_course_registration_template.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        $:_("You have been registered for the course: <strong>{course_name}</strong> at Universidad Nacional de Colombia.")
+        <br><br>
+        $:_("If you have already activated your account, you can access to UNCode to start the course.")
+        <br><br>
+
+        <i>$:_("How to proceed?")</i><br>
+        <ol>
+            <li>$:_("When you logged in, you could see the main page with the courses that you are enrolled")</li>
+            <li>$:_("Click on <strong>{course_name}</strong> option on the course list or click this link:")
+                <ul><li>$:_("{course_link}")</li></ul>
```
Comment (AndresFelipeCastellanos) [inline inginious/frontend/plugins/register_students/static/email_course_registration_template.html]: Leave this variable without the messages translation syntax. Also make sure to update de i18n files when change this and the above messages.

File: inginious/frontend/plugins/register_students/pages/api/add_course_students_csv_file_api.py [hunk 6]
```diff
@@ -72,14 +72,17 @@ def register_all_students(self, parsed_file, course, email_language):
         registered_on_course = 0
         registered_users = 0
         all_emails = []
```
Comment (AndresFelipeCastellanos) [inline inginious/frontend/plugins/register_students/pages/api/add_course_students_csv_file_api.py]: Remove this variable, as it is not used.

**Review by AndresFelipeCastellanos — CHANGES_REQUESTED** [review]
LGTM. There are some small comments.

**Review by AndresFelipeCastellanos — APPROVED** [review]

**Pull request merged**